js/util/config.js

```diff
@@ -1,6 +1,16 @@
 'use strict';
+// @flow
 
-module.exports = {
+type Config = {|
```

Comment:
It's the somewhat new "exact" object syntax. see: https://flowtype.org/docs/objects.html#exact-object-types